website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -208,10 +187,35 @@ There are a number of other useful configurations that can be made when initialz
 - Filter watch events using [predicates][event_filtering]
 - Choose the type of [EventHandler][event_handler_godocs] to change how a watch event will translate to reconcile requests for the reconcile loop. For operator relationships that are more complex than primary and secondary resources, the [`EnqueueRequestsFromMapFunc`][enqueue_requests_from_map_func] handler can be used to transform a watch event into an arbitrary set of reconcile requests.
 
+### Reconcile loop
 
-#### Reconcile loop
+The reconcile function is responsible for synchronizing the desired state as represented by their Specs and the actual state of the system. In this way, it works like a loop, and it does not stop until all conditionals match its implementation. 
 
-Every Controller has a Reconciler object with a `Reconcile()` method that implements the reconcile loop. The reconcile loop is passed the [`Request`][request-go-doc] argument which is a Namespace/Name key used to lookup the primary resource object, Memcached, from the cache:
+The following are a few possible return options to restart the Reconcile:
```

Comment:
It would be better to move the return values of `Reconcile()` below the description of `Reconcile()`, as it was before your changes.

Reply:
I did not get this one. I understand that it is as you suggested since we have: 

### Reconcile loop

The reconcile function is responsible for enforcing the desired CR state on the actual state of the system. It runs each time an event occurs on a watched CR or resource, and will return some value depending on whether those states match or not.

The following are a few possible return options for a Reconciler:

... examples....

If not, could you please help me understand your suggestion?